NeuroID/NIDParamsCreator.swift

```diff
@@ -180,7 +180,7 @@ enum ParamsCreator {
     static func getSDKVersion() -> String {
         // Version MUST start with 4. in order to be processed correctly
         let version = Bundle(for: NeuroIDTracker.self).object(forInfoDictionaryKey: "CFBundleShortVersionString") as? String
-        return "5.ios-\(version ?? "?")"
+        return "5.ios-\(NeuroID.isRN ? "rn-" : "")\(version ?? "?")"
```

Comment:
what does the 5. prefix stand for in ios and android?

Reply:
It was a requirement for collector. It determines if the traffic should be handled with new (v5) or old (v4) collector logic.